junit5/src/main/java/cz/xtf/junit5/extensions/OpenShiftRecorderService.java

```diff
@@ -215,6 +216,15 @@ public void recordState(ExtensionContext context) throws IOException {
                 !isMasterAndBuildNamespaceSame() ? getFilter(context, EVENT_FILTER_BUILDS) : null);
     }
 
+    public void recordState(ExtensionContext context, PrintStream ps) throws IOException {
+        savePodLogs(context, getFilter(context, POD_FILTER_MASTER),
+                !isMasterAndBuildNamespaceSame() ? getFilter(context, POD_FILTER_BUILDS) : null, ps);
+        saveBuildLogs(context, getFilter(context, BUILD_FILTER_MASTER),
+                !isMasterAndBuildNamespaceSame() ? getFilter(context, BUILD_FILTER_MASTER) : null, ps);
```

Comment:
please be aware of the same problem fixed in https://github.com/xtf-cz/xtf/pull/521